platform.js

```diff
@@ -119,11 +122,16 @@ module.exports = {
     tries -= 1;
 
     try {
-      let child = childProcess.spawn(command, args, Object.assign({}, opts, {
-        cwd: path.dirname(command),
+      const spawnOpts = Object.assign({}, opts, {
+        cwd: opts.cwd || path.dirname(command),
```

Comment:
The startProcess command now accepts a `cwd` override in the options, but does not allow callers to override stdio, or detached.